### PR TITLE
Initialize node-express

### PR DESCRIPTION
**Pull request by thearavind**

### CODE REVIEW

File: node-express/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    "env": {
+        "es6": true
+    },
+    "extends": "standard"
+};
```
Comment (ravi-lattis) [inline node-express/.eslintrc.js]: Leave a new line at the end pls.

File: node-express/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 3000
+CMD [ "npm", "start" ]
```
Comment (ravi-lattis) [inline node-express/Dockerfile]: Leave a new line at the end pls.

File: node-express/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+## TODO
+- [x] Docker config
+- [x] ESLint config
+- [x] Database connections
+- [ ] REST CRUD
+- [ ] Test
+
+## Run
+
+Navigate to the project folder ane build the docker image using the below command
```
Comment (ravi-lattis) [inline node-express/README.md]: Pls, correct the spelling for `ane` to `and`.

File: node-express/database/mongo.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const MongoClient = require('mongodb').MongoClient
+let _db
+
+/**
+ *  Connects to the MongoDB using the connection string
+ *  Visit http://mongodb.github.io/node-mongodb-native/ for the docs
+ *  @return {Promise} returns Promise which resolves the MongoDB connection Object
+ *  */
+async function connectToDB() {
+    _db = await MongoClient.connect(`mongodb://root:root@ds259865.mlab.com:59865/users`)
```
Comment (ravi-lattis) [inline node-express/database/mongo.js]: Get the database configuration from a central config file in the project.

File: node-express/database/mongo.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+/**
+ *  Getter function for the MongoDB Object
+ *  @return {object} returns the MongoDB connection object
+ *  */
+function getConnection() {
+    return _db
+}
+
+module.exports = {
+    connectToDB: connectToDB (),
```
Comment (ravi-lattis) [inline node-express/database/mongo.js]: There's a space came by mistake, I guess.

File: node-express/database/mysql.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+/**
+ *  Connects to the MySQL database and tests the connection is valid
+ *  Visit https://github.com/mysqljs/mysql for the docs
+ *  @return {Promise} returns Promise which resolves the MySQL connection Object
+ *  */
+function connectToDB() {
+    return new Promise((resolve) => {
+        _sql = mysql.createConnection({
+            host: 'localhost',
```
Comment (ravi-lattis) [inline node-express/database/mysql.js]: Get all the database configuration from a central config file.

File: node-express/package.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Levitators/JavaScript-Starter-Kit.git"
+  },
+  "keywords": [
+    "Boilerplate",
+    "NodeJS",
+    "Express"
+  ],
+  "author": "Aravindhan (thearuvi@gmail.com)",
```
Comment (ravi-lattis) [inline node-express/package.json]: Please change the author to `Levitators`.

File: node-express/.eslintrc.js (hunk 1 above)
Comment (ravirajsubramanian) [inline node-express/.eslintrc.js]: Please leave an empty line at the bottom.

File: node-express/Dockerfile (hunk 2 above)
Comment (ravirajsubramanian) [inline node-express/Dockerfile]: Please leave an empty line at the bottom.

File: node-express/README.md (hunk 3 above)
Comment (ravirajsubramanian) [inline node-express/README.md]: Correct the spelling for `ane` to `and`.

File: node-express/database/mongo.js (hunk 4 above)
Comment (ravirajsubramanian) [inline node-express/database/mongo.js]: Get all the database configuration details from a central config file.

File: node-express/database/mongo.js (hunk 5 above)
Comment (ravirajsubramanian) [inline node-express/database/mongo.js]: There's a space came by mistake, I guess.

File: node-express/database/pgsql.js [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+let _pg
+
+/**
+ *  Connects to the PostgreSQL database and tests the connection is valid
+ *  Visit http://vitaly-t.github.io/pg-promise/ for docs
+ *  @return {Promise} returns Promise which resolves the PostgreSQL connection Object
+ *  */
+function connectToDB() {
+    return new Promise((resolve) => {
+        _pg = pgp(`postgres://birlcvav:19QZsLOLiARafPCZOatrZobWlRYF5XGL@qdjjtnkv.db.elephantsql.com:5432/birlcvav`);
```
Comment (ravirajsubramanian) [inline node-express/database/pgsql.js]: Get all the database configuration details from a central config file. Also, Do not expose your personal database configurations.

File: node-express/index.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+app.use(bodyParser.urlencoded({extended: true}))
+app.use(bodyParser.json())
+app.use(express.static('public'))
+
+/*Start the app only if connected to the database
+* To use other databases import the respective files from the
+* database folder and follow the same steps as below*/
+mongo.connectToDB()
+    .then(() => {
+    let port = process.env.PORT || 3000
```
Comment (ravirajsubramanian) [inline node-express/index.js]: port can be `const`.

File: node-express/package.json (hunk 7 above)
Comment (ravirajsubramanian) [inline node-express/package.json]: Please change the author to `Levitators`.

**Review by ravirajsubramanian — CHANGES_REQUESTED** [review]

**Review by ravi-lattis — APPROVED** [review]

**Review by ravirajsubramanian — APPROVED** [review]

**Pull request merged**